lib/generators/solidus_multi_domain/install/install_generator.rb

```diff
@@ -6,19 +6,19 @@ class InstallGenerator < Rails::Generators::Base
       class_option :auto_run_migrations, type: :boolean, default: false
 
       def add_javascripts
-        append_file "vendor/assets/javascripts/spree/backend/all.js", "//= require spree/backend/solidus_multi_domain\n"
+        append_file 'vendor/assets/javascripts/spree/backend/all.js', "//= require spree/backend/solidus_multi_domain\n"
```

Comment:
no need to install empty files on host applications, let's remove this method.

Reply:
The file added as a dependency has a `require_tree` that is not very visible and you might have missed 🔍